_sass/_navigation.scss

```diff
@@ -35,8 +35,7 @@ ul, ul ul {
 }
 .menu > li,
 .menu > li > a,
-.menu > li > ul > li,
-.menu > li > ul > li > ul > li {
+.menu > li > ul > li{
```

Comment:
CSS clean up (removed unused entries). Other uncommented changes in this file are also clean up.